CHANGELOG.unreleased.md

```diff
@@ -8,6 +8,7 @@
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
 - [VM/Network] Show IP addresses in front of their VIFs [#4882](https://github.com/vatesfr/xen-orchestra/issues/4882) (PR [#5003](https://github.com/vatesfr/xen-orchestra/pull/5003))
+- [Home/VM] Improve the (not) backed up VMs list (PR [#5046](https://github.com/vatesfr/xen-orchestra/pull/5046)
```

Comment:
This is vague and does not bring any information to users.

Either be more specific or remove it entirely if you are unable to.